doc/source/news.rst

```diff
@@ -3,6 +3,34 @@
 News
 ====
 
+.. _release-12-07:
+
+Release 12.07 - 2022-08-30
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/install/centos`][:doc:`/install/almalinux`] Added support for MariaDB 10.3.36.
+
```

Comment:
We should write the following modification.

* https://github.com/mroonga/mroonga/commit/050d8e8db5d52229b91880b5067ce8b6c366cefe
* https://github.com/mroonga/mroonga/commit/8599dbc65608920cc6200874bd27bc3dd2f390b2
* https://github.com/mroonga/mroonga/commit/5e8cf1d1e7d773890fa0ee034431eb3aa3f11604
* https://github.com/mroonga/mroonga/commit/21bc19685d7605210a9f4d1b1527149ae638b640

Reply:
>     * [050d8e8](https://github.com/mroonga/mroonga/commit/050d8e8db5d52229b91880b5067ce8b6c366cefe)

As I mentioned on the description, I have intentionally omitted this change because Percona Server 8.0.29 may have critical problem.
Should we add this change for news?

>     * [8599dbc](https://github.com/mroonga/mroonga/commit/8599dbc65608920cc6200874bd27bc3dd2f390b2)

I have added this as Fixes.
Should we add this as Improvements?

>     * [5e8cf1d](https://github.com/mroonga/mroonga/commit/5e8cf1d1e7d773890fa0ee034431eb3aa3f11604)

Ah, I missed an added support for Prcona Server 5.7.39-42.

>     * [21bc196](https://github.com/mroonga/mroonga/commit/21bc19685d7605210a9f4d1b1527149ae638b640)

Also I missed changes after mariadb-10.3-mroonga.spec.in...

Reply:
> As I mentioned on the description, I have intentionally omitted this change because Percona Server 8.0.29 may have critical problem.
> Should we add this change for news?

Ah...Sorry, I forgot it...
You are right. 050d8e8 should not add to the news.

Reply:
Is the explanation of 8599dbc "Fixed a bug that Mroonga returns results with a wrong character code."?

Reply:
> Is the explanation of https://github.com/mroonga/mroonga/commit/8599dbc65608920cc6200874bd27bc3dd2f390b2 "Fixed a bug that Mroonga returns results with a wrong character code."?

Yes, ...is my understanding wrong?

Reply:
I have fixed description about this item according to https://github.com/mroonga/docker/issues/75